apps/token/src/routes/staking/node-list.tsx

```diff
@@ -97,140 +88,105 @@ export const NodeList = ({ epoch, party }: NodeListProps) => {
           ? '-'
           : stakedOnNode.dividedBy(stakedTotal).times(100).dp(2).toString() +
             '%';
-
-      const userStake = party?.delegations?.length
-        ? party?.delegations
-            ?.filter((d) => d.node.id === node.id)
-            ?.filter((d) => d.epoch === Number(epoch?.id))
-            .reduce((sum, d) => {
-              const value = new BigNumber(d.amountFormatted);
-              return sum.plus(value);
-            }, new BigNumber(0))
-        : new BigNumber(0);
-
-      const userStakePercentage =
-        userStake.isEqualTo(0) || stakedOnNode.isEqualTo(0)
-          ? '-'
-          : userStake.dividedBy(stakedOnNode).times(100).dp(2).toString() + '%';
+      const status = t(`status-${node.rankingScore.status}`);
 
       return {
         id: node.id,
-        name: node.name,
-        pubkey: node.pubkey,
-        stakedTotal,
-        stakedOnNode,
-        stakedTotalPercentage,
-        userStake,
-        userStakePercentage,
-        epoch,
-        scores: node.rankingScore,
+        [t('validator')]: {
+          avatarUrl: node.avatarUrl,
+          name: node.name,
+        },
+        [t('status')]: status,
+        [t('totalStakeThisEpoch')]: formatNumber(stakedTotal, 2),
+        [t('share')]: stakedTotalPercentage,
+        [t('validatorStake')]: formatNumber(stakedOnNode, 2),
+        [t('nextEpoch')]: node.pendingStake,
+        [t('rankingScore')]: formatNumber(
```

Comment:
suggestion (non blocking): Might be good to switch to using formatters from libs/react-helpers